pandas/io/excel/_base.py

```diff
@@ -902,6 +913,12 @@ def __init__(
         self, path_or_buffer, engine=None, storage_options: StorageOptions = None
     ):
         if engine is None:
+            warnings.warn(
+                "The default argument engine=None is deprecated. "
+                "Specify the engine argument to suppress this warning.",
```

Comment:
I think we need to clarify this message, to explain the reason why it is deprecated, and what people can do: either specify "xlrd", or either install openpyxl and specify that

Reply:
Changing to the following. Let me know of any thoughts.

> DeprecationWarning: The default argument engine=None is deprecated. Using None defaults to the xlrd engine which is no longer maintained. The default value will be 'openpyxl' in a future version of pandas, although xlrd will continue to be allowed for the indefinite future. Either install openpyxl and specify it as the engine, or specify 'xlrd' to suppress this warning.

Reply:
For the current behaviour, that sounds good!